src/main/java/archibald/Archibald.java

```diff
@@ -0,0 +1,206 @@
+package archibald;
+import java.util.Scanner;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+import archibald.task.Deadline;
+import archibald.task.Event;
+import archibald.task.Task;
+import archibald.task.Todo;
+
+class ArchibaldException extends Exception {
+    public ArchibaldException(String message) {
+        super(message);
+    }
+}
+
+public class Archibald {
+
+    private static List<Task> tasks = new ArrayList<>();
+
+    public static void printArchibaldResponse(String message) {
+        System.out.println("____________________________________________________________");
+        System.out.println(message);
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void addTask(String input) {
+        try {
+            String[] parts = input.split(" ", 2);
+            String type = parts[0].toLowerCase();
+    
+            if (type.equals("todo") || type.equals("deadline") || type.equals("event")) {
+                if (parts.length < 2 || parts[1].trim().isEmpty()) {
+                    throw new ArchibaldException("Error: By royal decree, the description of a " + type + " cannot be empty.");
+                }
+            }
+    
+            Task newTask;
+            switch (type) {
+                case "todo":
+                    newTask = new Todo(parts[1]);
+                    break;
+                case "deadline":
+                    String[] deadlineParts = parts[1].split(" /by ", 2);
```

Comment:
Try to avoid magic literals, you can consider using named constants to give the literals some meaning so that it will be easier to understand as well. 
E.g.
```
static final double PI = 3.14236;
static final int MAX_SIZE = 10;
...
return PI;
...
return MAX_SIZE - 1;

```